runner/operator/roslin_operator/roslin_operator.py

```diff
@@ -71,6 +71,10 @@ def get_jobs(self):
                 sample_mapping += "\t".join(
                     [normal_sample_name, FileProcessor.parse_path_from_uri(p['location'])]) + "\n"
 
+            for p in job['pair'][1]['bam']:
+                sample_mapping += "\t".join(
+                    [normal_sample_name, FileProcessor.parse_path_from_uri(p['location'])]) + "\n"
```

Comment:
Now I see we are using it here. We should try doing this everywhere we have path manipulation.